framework/encode/vulkan_handle_wrapper_util.h

```diff
@@ -140,7 +205,12 @@ void CreateWrappedNonDispatchHandle(typename Wrapper::HandleType* handle, PFN_Ge
         Wrapper* wrapper   = new Wrapper;
         wrapper->handle    = (*handle);
         wrapper->handle_id = get_id();
-        (*handle)          = reinterpret_cast<typename Wrapper::HandleType>(wrapper);
+        if (!state_handle_table_.InsertWrapper(wrapper))
```

Comment:
Under what circumstance would this happen?  Would we have to have an internal error, like finding the wrapper for the handle failed in a parent function, but then inserting the wrapper here in this function failed because the handle was already in the map?

Reply:
It could happen in multi-threading. When it destroys and creates by mutli-threading at the same time, it could create the same value of the new handle, and might not be able to erase the destroyed data in time before inserting a new one.